public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -642,9 +642,18 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
     return this._serializer.initialSaveModel;
   }
 
+  public getInitialVersion = () => {
+    return this._serializer.getInitialVersion();
+  };
+
+  public getSnapshotUrl = () => {
+    return this._serializer.getSnapshotUrl();
+  };
+
   /** Hacky temp function until we refactor transformSaveModelToScene a bit */
-  public setInitialSaveModel(saveModel?: Dashboard | DashboardV2Spec) {
+  public setInitialSaveModel(version: number | undefined, saveModel?: Dashboard | DashboardV2Spec) {
```

Comment:
Why don't we just save the metadata from the schema? We're losing the uid or the folder for the dashboard, for example.

Actually, the scene already save the meta. Can't we use the version from there?

Reply:
Ah you are right, not sure why I assumed that meta in state won't be updated after save (and it actually isn't, think it's a bug. the initial save model is updated, but not the meta).. Think we should be able to use it.

Reply:
actually, version is part of the meta, but it was not exposed in the `DashboardMeta` type, i've added it and updated the implementation. However, let's talk today about if it's even needed, i have doubts about `ignoreChanges` implementation.